jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -1054,9 +1054,8 @@ def _load_regions(self, regions, combination_mode=None, max_num_regions=None,
         combination_mode : list, str, or `None`
             The way that regions are created or combined. If a list, then it must be the
             same length as regions. Each element describes how the corresponding region
-            in regions will be applied. If `None`, then it will follow the default glue
-            functionality for subset creation. Options are ['new', 'replace', 'or', 'and',
-            'xor', 'andnot']
+            in regions will be applied. If `None`, then a new subset will be created.
+            Options are ['new', 'replace', 'or', 'and', 'xor', 'andnot']
```

Comment:
I think similar changes need to be added to the docstring for the user-facing `import_region`.  Can we also mention somewhere that this will not result in a change to the combination_mode selection, but _will_ result in changing the selected subset to the new subset?